pkg/metrics/metrics.jsx

```diff
@@ -479,7 +478,13 @@ class CurrentMetrics extends React.Component {
                             <DescriptionList className="pf-m-horizontal-on-sm">
                                 <DescriptionListGroup>
                                     <DescriptionListTerm>{ _("Load") }</DescriptionListTerm>
-                                    <DescriptionListDescription id="load-avg">{this.state.loadAvg}</DescriptionListDescription>
+                                    <DescriptionListDescription id="load-avg">
+                                        <Flex>
+                                            <FlexItem spacer={{ default: 'spacerSm' }}>{ _("1 min") }:&nbsp;{ this.state.loadAvg[0] },</FlexItem>
```

Comment:
It's a bit stupid having to repeat the spacer= attribute, instead of setting `<Flex spaceItems={{ modifier: 'spaceItemsSm' }}>` as the documentation suggests. But the latter just simply doesn't work. @KKoukiou , @marusak , do you happen to have any idea why?

Reply:
I think you just use it incorrectly.
`spaceItems={ default: 'spaceItemsXs' }` is the correct usage.

Reply:
Right:
``` diff
diff --git pkg/metrics/metrics.jsx pkg/metrics/metrics.jsx
index 376ed0e27..5bb0c6e35 100644
--- pkg/metrics/metrics.jsx
+++ pkg/metrics/metrics.jsx
@@ -479,10 +479,10 @@ class CurrentMetrics extends React.Component {
                                 <DescriptionListGroup>
                                     <DescriptionListTerm>{ _("Load") }</DescriptionListTerm>
                                     <DescriptionListDescription id="load-avg">
-                                        <Flex>
-                                            <FlexItem spacer={{ default: 'spacerSm' }}>{ _("1 min") }:&nbsp;{ this.state.loadAvg[0] },</FlexItem>
-                                            <FlexItem spacer={{ default: 'spacerSm' }}>{ _("5 min") }:&nbsp;{ this.state.loadAvg[1] },</FlexItem>
-                                            <FlexItem spacer={{ default: 'spacerSm' }}>{ _("15 min") }:&nbsp;{ this.state.loadAvg[2] }</FlexItem>
+                                        <Flex spaceItems={{ default: 'spaceItemsXs' }}>
+                                            <FlexItem>{ _("1 min") }:&nbsp;{ this.state.loadAvg[0] },</FlexItem>
+                                            <FlexItem>{ _("5 min") }:&nbsp;{ this.state.loadAvg[1] },</FlexItem>
+                                            <FlexItem>{ _("15 min") }:&nbsp;{ this.state.loadAvg[2] }</FlexItem>
                                         </Flex>
                                     </DescriptionListDescription>
                                 </DescriptionListGroup>
```

Reply:
Hah, thank you! That was a bit confusing in the docs.